### PR TITLE
Switch to github actions

### LINKED ISSUE #5: Migrate Travis -> Github Actions
Travis will be limited open source projects build time soon, so we need to transition to github actions. 

Take a look at how we did it [open discussions](https://github.com/mitodl/open-discussions/pull/3204) or [bootcamps](https://github.com/mitodl/bootcamp-ecommerce/pull/1086) or [micromasters](https://github.com/mitodl/micromasters/pull/4682)

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #5

### CODE REVIEW

**Comment by rhysyngsun** [comment]
@noisecapella looks like it still failed?

**Comment by noisecapella** [comment]
@rhysyngsun got it figured out, ready for review now

File: requirements.txt [hunk 1]
```diff
@@ -27,7 +27,7 @@ factory-boy==3.0.1        # via -r requirements.in
 faker==4.1.2              # via -r requirements.in, factory-boy
 idna==2.10                # via requests
 ipython-genutils==0.2.0   # via traitlets
-ipython==7.17.0           # via -r requirements.in
+ipython==7.19.0           # via -r /home/george/Projects/ocw-studio/ocw_studio/ocw_studio/requirements.in
```
Comment (rhysyngsun) [inline requirements.txt:30]: This has your full local path in it

**Review by rhysyngsun — CHANGES_REQUESTED** [review]
One small thing

**Comment by noisecapella** [comment]
@rhysyngsun I ran pip-compile again and that should make my local path go away. However there are lots of indentation changes. I think that's just what pip-compile does now as their example on the readme at https://github.com/jazzband/pip-tools is formatted the same way

**Review by rhysyngsun — APPROVED** [review]

**Pull request merged**